addons/mrp/models/stock_move.py

```diff
@@ -242,6 +242,10 @@ def _should_be_assigned(self):
         res = super(StockMove, self)._should_be_assigned()
         return bool(res and not (self.production_id or self.raw_material_production_id))
 
+    def _should_bypass_reservation(self):
+        res = super(StockMove, self)._should_bypass_reservation()
+        return bool(res and not (self.production_id or self.raw_material_production_id))
```

Comment:
move with raw_material_production_id are the porduction component. So they need to reserve quantity normally. 
Only those with `production_id` should not call action_assign